renderer.md

```diff
@@ -0,0 +1,30 @@
+## Disclaimer
+
+First of all this is an attempt to gather information about state of Renderer in dlang
+
+# What is a Renderer?
+ 
+Renderer contains a rendering state and provides functionality to draw primitives using specific color or texture. Renderer uses Image as a buffer to render to. Calling renderer method issues command to underlying hardware. The command is queued and flushed at some later time using some form of `flush` method. Renderer draw the following primitives:
+- point
+- line
+- polyline
+- polygone (probably convex only?)
+- rectangle
+- ellipse
+- text
+- image
+
+Lines can be stroked, 2D figures can be stroked and filled. Rectange can have rounded corners.
+
+Renderer is 2D only.
+Renderer is rasterizing. Vector renderer is out of scope this document.
```

Comment:
This line is confusing, the text above and the primitive operations describe a vector renderer... but then this line?